lib/inspec/cli.rb

```diff
@@ -194,6 +194,8 @@ def detect
     desc: 'A single command string to run instead of launching the shell'
   option :format, type: :string, default: nil, hide: true,
     desc: 'Which formatter to use: cli, documentation, html, json, json-min, junit, progress'
+  option :depends, type: :array, default: [],
+    desc: 'A list of local folders that are loaded as dependencies (e.g. for resources)'
```

Comment:
Proposed alteration to make it clear what type of folder we're expecting:

`'A space-delimited list of local folders containing profiles whose libraries and resources will be loaded into the new shell'`

Reply:
Much nicer words, thank you, i'll add it!